app/graphql/mutations/visit_street_art.rb

```diff
@@ -0,0 +1,12 @@
+class Mutations::VisitStreetArt < Mutations::BaseMutation
+  argument :visited, Boolean, required: true
+  argument :street_art_id, Integer, required: true
+
+  type Types::StreetArtType
+
+  def resolve(visited:, street_art_id:)
+    art = StreetArt.find(street_art_id)
+    art.update(visited: visited)
+    art.reload
```

Comment:
thanks for this tip

Reply:
I had only used this in tests before but it is definitely clutch!

Reply:
yeah had some issues in mod 2 i think with testing not knowing what it just did